pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -186,71 +188,97 @@ def images() -> DownloadableImageCollection:
     return DownloadableImageCollection({**content_images, **style_images})
 
 
+class SkipSmallIterableImageFolderDataset(torch.utils.data.IterableDataset):
```

Comment:
Not sure why you want to subclass here. Composition should be way easier. Why not simply do

```py
dataset = SkipSmallImagesDataset(ImageFolderDataset(root), transform=transform, ...)
```

where `SkipSmallImagesDataset` is `Dataset` from https://github.com/pystiche/papers/pull/286#issuecomment-1082832974?